theme/templates/resource-landing-page/modals.html

```diff
@@ -1775,7 +1775,7 @@ <h4 class="modal-title" id="rep-res-to-irods-title">Copy resource bag to your iR
                 </div>
                 <div class="modal-footer">
                     <button type="button" class="btn btn-default" data-dismiss="modal">Cancel</button>
-                    <a id="btn-replicate" class="btn btn-danger">Copy</a>
+                    <a id="btn-replicate" class="btn btn-success">Copy</a>
```

Comment:
Was this intentional?

Reply:
Yes, I thought there is no point to make the button red in danger style since copying resource bag to user zone is not disruptive or a danger operation like deletion. Changing it to success button style will make it green as in other similar cases such as resource copy.